config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,14 @@
+CarrierWave.configure do |config|
```

Comment:
Please delete `public/uploads/photo.png` and add `public/uploads` to your git ignore!